src/helpers/print.js

```diff
@@ -50,7 +50,7 @@ export const print = async(dispatch, isEmbedPrintSupported, sortStrategy, colorM
       pagesToPrint,
       includeComments,
       includeAnnotations,
-      printQuality,
+      printQuality||PRINT_QUALITY,
```

Comment:
Better have default assignment in line 20, as in `({ includeAnnotations, includeComments, pagesToPrint, onProgress, printWithoutModal, printQuality = PRINT_QUALITY} = options)`